bciers/apps/compliance/src/tests/components/compliance-summary/request-issuance/track-status-of-issuance/IssuanceStatusApprovedNote.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { IssuanceStatusApprovedNote } from "../../../../../app/components/compliance-summary/request-issuance/track-status-of-issuance/IssuanceStatusApprovedNote";
+import { vi } from "vitest";
+import React from "react";
+import { render, screen } from "@testing-library/react";
+
+vi.mock("@bciers/components/icons/Check", () => ({
+  __esModule: true,
+  default: (props: any) => (
+    <div data-testid="check-icon" {...props}>
+      Check Icon
+    </div>
+  ),
+}));
+
+describe("IssuanceStatusApprovedNote", () => {
```

Comment:
We should also verify that the correct BCCR href is being provided in this alert note.